liquidity-rewards/contracts/tests/ERC20Mock.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+
+// mock class using ERC20
+contract ERC20Mock is ERC20 {
```

Comment:
Instead of having this file, can we import it from https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/mocks/ERC20Mock.sol?